ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -160,14 +161,13 @@ public int execute() {
     }
   }
 
-  private boolean shouldLoadAuthorizationMetadata() {
-    return conf.getBoolVar(HiveConf.ConfVars.REPL_INCLUDE_AUTHORIZATION_METADATA);
-  }
-
-  private void initiateAuthorizationLoadTask() throws SemanticException {
+  private void initiateRangerLoadTask() throws SemanticException {
```

Comment:
this will also try and replicate ranger policies unnecessarily. Just adding a deny policy should be sufficient.

Reply:
if REPL_INCLUDE_AUTHORISATION_METADATA is not enabled, rangerLoadRoot would be set to null and RangerLoadTask would not replicate ranger policies if rangerLoadRoot is null. It would only create a deny policy for target db in this case.